templates/selenium3.141.x-python37/src/__ALGO__.py

```diff
@@ -12,20 +13,26 @@ def get_chrome():
     return driver
 
 def get_phantomjs():
-    driver = webdriver.PhantomJS()
+    # PhantomJS must put logs in a place where it has write permissions
+    driver = webdriver.PhantomJS(service_log_path="/tmp/ghostdriver.log")
     return driver
 
 # API calls will begin at the apply() method, with the request body passed as 'input'
 # For more details, see algorithmia.com/developers/algorithm-development/languages
 def apply(input):
+    # note: you MUST have internet access enabled for this test work
```

Comment:
I'm still not a huge fan of this. I would prefer this wasn't the case. Maybe we can add something to jira about making package sets that need internet so at least the default internet permission would be correct. @kennydaniel might have thoughts on that.

An alternative would be to spin up a local server and use that but that might look gross and not too useful